spec/translation_quality/checks/glossary_compliance.yaml

```diff
@@ -18,11 +18,28 @@ components:
           maximum: 100
           example: 10
 
+    ProfilesGlossaryComplianceSettingsDTO:
+      allOf:
+        - $ref: '#/components/schemas/GlossaryComplianceSettingsDTO'
+        - type: object
+          properties:
+            ignoreSourceCount:
+              type: boolean
+              description: Ignore source count.
```

Comment:
Need a better description of the toggle.
By default this option is disabled.
If enabled then if glossary term appears in source 2 times then we expect that in translation translated terms will appear 2 times too.  Not sure how to say about it better.
Need help of @bbristow-smartling